content/news/2020-12-01-about-me.md

```diff
@@ -0,0 +1,74 @@
+title: All about my Outreachy internship with Mixxx DJ: Intro
+authors: Aanyu Deborah Oduman
+tags: website, Outreachy, internhip
+date: 2020-12-01 23:15:00
+## A little bit about me.
+
+<table>
+  <tr>
+     <td></td>
+     <td></td>
+  </tr>
+  <tr>
+    <td><img src="/images/news/hello.jpg" width=450 height=360></td>
```

Comment:
should we create a subfolder for the images so news/ doesn't get crowded?  like images/news/2020-12-01-about-me/ @Holzhaus

Reply:
I thought about that as well. But let keep things consistent and not switch to another naming scheme in this PR. I'll try to come up with a proposal and open a new PR.